sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeContentFromUrlAsync.java

```diff
@@ -32,7 +32,7 @@ public static void main(final String[] args)  {
             .buildAsyncClient();
 
         PollerFlux<FormRecognizerOperationResult, List<FormPage>> recognizeContentPoller =
-            client.beginRecognizeContentFromUrl("https://raw.githubusercontent.com/Azure/azure-sdk-for-java/master/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/sample-forms/forms/layout1.jpg");
+            client.beginRecognizeContentFromUrl("https://raw.githubusercontent.com/Azure/azure-sdk-for-java/master/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/sample-forms/forms/Form_1.jpg");
```

Comment:
is this intended?

Reply:
yeah, this file no longer exists so updated the sample.